### PR TITLE
Fix/adf 12/code integration

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/ADF-12

### CODE REVIEW

**Review by GregoryGagua — APPROVED** [review]

File: model/search/ResultsWatcher.php [hunk 1]
```diff
@@ -15,12 +15,13 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2018 (original work) Open Assessment Technologies SA;
- *
+ * Copyright (c) 2018-2020 (original work) Open Assessment Technologies SA;
```
Comment (mike-ionut-mihai-sandu-tao) [inline model/search/ResultsWatcher.php:18]: 2021 ?

**Review by mike-ionut-mihai-sandu-tao — COMMENTED** [review]
lgtm , let's check the pipeline

File: model/search/ResultsWatcher.php [hunk 2]
```diff
@@ -114,4 +131,23 @@ public function isResultSearchEnabled()
     {
         return (bool)$this->getOption(self::OPTION_RESULT_SEARCH_FIELD_VISIBILITY);
     }
+
+    private function transformDateTime(string $getStartTime): string
+    {
+        $timeArray = explode(" ", $getStartTime);
```
Comment (shpran) [inline model/search/ResultsWatcher.php:137]:
```suggestion
        $timeArray = explode(' ', $getStartTime);
```

**Review by shpran — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**